### ISSUE TITLE
Crossing: When switching to crossing=unmarked, remove crossing:markings=zebra etc

### ISSUE THREAD

**Issue by tordans** [openstreetmap/id-tagging-schema#1283]
This is just a stub for now. I just noticed the tagging was not removed. We should check this after the crossing refactoring was merged.  Not sure if there is a good way to build this, since not all tags should be removed, just some values.

**Comment by k-yle** [comment]
@tordans this is possible using `removeTags`, for example:

https://github.com/openstreetmap/id-tagging-schema/blob/331f51da8b10d62c3babb3f3d0acce9bfb76e489/data/presets/highway/construction.json#L20-L24

**Comment by tordans** [comment]
@k-yle right, but I was wondering if we really want to remove all tags or just some, when "leaving" the preset. Personally I will wait for https://github.com/openstreetmap/id-tagging-schema/pull/1201 to be merged to have a clean slate on alls things crossing.

https://github.com/ideditor/schema-builder?tab=readme-ov-file#removetags

**Comment by 1ec5** [comment]
I assume this is actually about overwriting `crossing:markings=*` with `crossing:markings=no`? Otherwise, it would immediately trigger a validator warning about incomplete tagging.

Also, for consideration: `crossing:markings=surface` answers the question, “Are there painted markings?” with, “Well, not really, but there is a surface treatment.” In some jurisdictions, a surface treatment isn’t legally considered a marking, and there’s still debate among specialists about whether they’re more or less visible than standard markings.

The `crossing:markings=*` proposal included this value with the understanding that some laypeople wouldn’t know that, pedantically, it doesn’t count as an unmarked crossing, so at least it should be recorded somehow. The Unmarked Crossing preset doesn’t include a field for this tag, but I suspect we should preserve it given the lack of real-world clarity.

**Comment by matkoniecz** [comment]
I expected https://github.com/openstreetmap/iD/pull/11696 to resolve it, but apparently for some reason does not.

**Comment by matkoniecz** [comment]
I have now tested it in some other way and it worked. Well, lets get specific

changing 

```
crossing=uncontrolled
crossing:island=yes
highway=crossing
tactile_paving=yes
crossing:markings=aaaa
```

to Unmarked Crossing preset, in Germany, worked

Does anyone has case where it does not work as expected?

**Comment by matkoniecz** [comment]
Though I run into an opposite problem: https://github.com/openstreetmap/id-tagging-schema/issues/2606
